styles.css

```diff
@@ -3,17 +3,18 @@
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box;
+  /* box-sizing: border-box; */
```

Comment:
- [ ] [REQURED] Please, remove commented code. Because it polluted the codebase. It's better to use `git reset` to come back to previous versions ◀️.